service/identity.go

```diff
@@ -92,3 +93,105 @@ func (s *service) Probe(
 	log.Debugf(fmt.Sprintf("Probe returning: %v", rep.Ready.GetValue()))
 	return rep, nil
 }
+
+func (s *service) ProbeController(ctx context.Context,
```

Comment:
can we move this method to service.go where we have all the probe methods defined.